# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xulrunner/gcc43.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/client/linux/handler/minidump_generator.cc.orig	2009-12-21 19:34:00.000000000 -0330
+++ mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/client/linux/handler/minidump_generator.cc	2010-11-17 15:15:53.000000000 -0330
@@ -37,6 +37,7 @@
 #include <unistd.h>
 #include <sys/utsname.h>
 #include <sys/wait.h>
+#include <stdio.h>
 
 #include <cstdlib>
 #include <ctime>
--- mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/client/linux/handler/exception_handler.cc.orig	2009-12-21 19:34:00.000000000 -0330
+++ mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/client/linux/handler/exception_handler.cc	2010-11-17 15:15:27.000000000 -0330
@@ -33,6 +33,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <stdio.h>
 
 #include <cassert>
 #include <cstdlib>
--- mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/common/linux/dump_symbols.cc.orig	2009-12-21 19:34:01.000000000 -0330
+++ mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/common/linux/dump_symbols.cc	2010-11-17 13:52:51.000000000 -0330
@@ -41,6 +41,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <algorithm>
+#include <stdio.h>
 
 #include <functional>
 #include <list>
--- mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/common/linux/file_id.cc.vanilla	2010-11-19 16:25:56.000000000 +0100
+++ mozilla-1.9.1/toolkit/crashreporter/google-breakpad/src/common/linux/file_id.cc	2010-11-19 16:26:50.000000000 +0100
@@ -39,6 +39,8 @@
 #include <link.h>
 #include <sys/mman.h>
 #include <string.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 #include <unistd.h>
 
 #include "common/linux/file_id.h"
